diff --git a/node_modules/@project-serum/anchor/dist/cjs/index.js b/node_modules/@project-serum/anchor/dist/cjs/index.js
index f30f23d..6dfb788 100644
--- a/node_modules/@project-serum/anchor/dist/cjs/index.js
+++ b/node_modules/@project-serum/anchor/dist/cjs/index.js
@@ -27,6 +27,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.utils = exports.setProvider = exports.getProvider = exports.Provider = exports.web3 = exports.BN = void 0;
 const nodewallet_1 = __importDefault(require("./nodewallet"));
+Object.defineProperty(exports, "Wallet", { enumerable: true, get: function () { return nodewallet_1.default; } });
 const common_js_1 = require("./utils/common.js");
 var bn_js_1 = require("bn.js");
 Object.defineProperty(exports, "BN", { enumerable: true, get: function () { return __importDefault(bn_js_1).default; } });
